src/index.ts

```diff
@@ -30,6 +31,7 @@ export class Authenticator {
   _disableCookieDomain: boolean;
   _httpOnly: boolean;
   _sameSite?: SameSite;
+  _enableLogout?: boolean;
```

Comment:
I would recommend implementing a dedicated LogoutConfiguration. If it is not defined, the logout functionality would be disabled. 

```ts
interface LogoutConfiguration {
  logoutUri: string;
  logoutRedirectUri: string;
}
```